docs/_docs/continuous-integration/circleci.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "CircleCI"
+---
+
+Building, testing, and deploying your Jekyll generated website can quickly be done with [CircleCI][0]. [GitHub][1] and [Bitbucket][2] are supported and you can get started with an open-source or private repository for free.
+
+[0]: https://circleci.com/
+[1]: https://github.com/
+[2]: https://bitbucket.org/
+
+## Follow Your Project on CircleCI
+
+To start building your project on CircleCI, all you need to do is 'follow' your project from CircleCI's website:
+
+1. Visit the 'Add Projects' page: <https://circleci.com/add-projects>.
+1. From the GitHub or Bitbucket tab on the left, choose a user or organization.
+1. Find your project in the list and click 'Build project' on the right.
```

Comment:
This is a shorthand for many versions of Markdown. You just put all `1.`s but when rendered, Markdown will put the proper number. I like doing it this way so that in the future, if you need to add more items in between or switch the order of items, you don't need to worry about changing the numbers too. Less maintenance.